packages/story-editor/src/components/canvas/editLayer.js

```diff
@@ -33,11 +39,12 @@ import withOverlay from '../overlay/withOverlay';
 import EditElement from './editElement';
 import { Layer, PageArea, FooterArea, Z_INDEX } from './layout';
 import useFocusCanvas from './useFocusCanvas';
+import SingleSelectionMoveable from './singleSelectionMoveable';
 
-const LayerWithGrayout = styled(Layer)`
+const LayerWithGrayout = withOverlay(styled(Layer)`
   background-color: ${({ grayout, theme }) =>
     grayout ? theme.colors.opacity.overlayDark : 'transparent'};
-`;
+`);
 
 const EditPageArea = withOverlay(PageArea);
```

Comment:
I'm not sure if both elements need the overlay thingy here. @miina, do you know? It seems like it is required, so I will keep it for now.

Reply:
Not sure either, if it was previously for `Moveable` only then it shouldn't be needed anymore -- needs testing.